src/foam/nanos/notification/NotificationSetting.js

```diff
@@ -120,7 +120,7 @@ foam.CLASS({
       ],
       type: 'String',
       javaCode: `
-        return "notificationsetting." + operation + "." + getId();
+        return "notificationsetting." + operation + getId();
```

Comment:
I think this was correct before. Without the dot we end up with permissions like "notificationsetting.read1496"